cli/fossilize_replay_linux.hpp

```diff
@@ -1498,14 +1500,17 @@ static int run_slave_process(const VulkanDevice::Options &opts,
 	int ret = run_normal_process(replayer, databases, nullptr, 0, Global::metadata_fd);
 	global_replayer = nullptr;
 
-	// Cannot reliably handle these signals if they occur during teardown of the process.
```

Comment:
Any reason for moving the code?

Reply:
Sorry, it was by accident. I was originally trying to make the signalstack conditional too, then realized it wouldn't work with the worker threads. So I moved it outside but forgot about the shuffling...